services/src/main/java/io/scalecube/services/ServiceRegistryImpl.java

```diff
@@ -115,7 +119,7 @@ private void loadMemberServices(DiscoveryType type, Member member) {
   /**
    * register a service instance at the cluster.
    */
-  public void registerService(Object serviceObject) {
+  public void registerService(Object serviceObject, Tag[] tags) {
```

Comment:
maybe it can be clean if you will use it consistently, but it is used interchangeably in different places with the Map of string so it was difficult for me to follow the logic when review